### PR TITLE
MDK5 sample project

### PR DESCRIPTION
**Pull request by kojo1**
with wrap_test.c

### CODE REVIEW

File: IDE/MDK5/dummy.c [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+int   TPM2_IoCb(TPM2_CTX* ctx, const byte* txBuf, byte* rxBuf,
+word16 xferSz, void* userCtx){
+	(void) ctx;
+	(void) txBuf;
+	(void) rxBuf;
+	(void)xferSz;
+	(void)userCtx;
+	return 0;
+}
+
```
Comment (dgarske) [inline IDE/MDK5/dummy.c]: @kojo1, if you'd like to plug in the STM32 CubeMX HAL SPI code its here:

https://github.com/wolfSSL/wolfTPM/blob/master/examples/tpm_io.c#L34
https://github.com/wolfSSL/wolfTPM/blob/master/examples/tpm_io.c#L60
https://github.com/wolfSSL/wolfTPM/blob/master/IDE/OPENSTM32/Src/main.c#L210

File: IDE/MDK5/RTE/wolfSSL/user_settings.h [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
```
Comment (dgarske) [inline IDE/MDK5/RTE/wolfSSL/user_settings.h]: Can you update this to match the other wolfTPM license headers (2018 and wolfTPM).

Comment (dgarske) [reply IDE/MDK5/RTE/wolfSSL/user_settings.h]: Also can you make sure `WOLF_CRYPTO_DEV` is added in here to support the wolfCrypt callbacks?

**Comment by kojo1** [comment]
UART4 is changed to USART1, just it does not seem to be defined with some STM32F4

**Comment by dgarske** [comment]
Replaced with PR #22.

**Pull request closed without merging**